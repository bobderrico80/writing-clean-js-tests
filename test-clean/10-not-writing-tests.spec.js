const complicatedLegacyModule = require('../src/complicatedLegacyModule');

describe('The complicatedLegacyModule', () => {
  it('doComplicatedTask() function works as expected', () =>
    expect(complicatedLegacyModule.doComplicatedTask('foo')).toMatchSnapshot());

  it('doAnotherComplicatedTask() function works as expected', () =>
    expect(complicatedLegacyModule.doAnotherComplicatedTask('bar')).toMatchSnapshot());
});
