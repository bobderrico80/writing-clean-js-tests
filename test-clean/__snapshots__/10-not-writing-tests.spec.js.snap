// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The complicatedLegacyModule doAnotherComplicatedTask() function works as expected 1`] = `
Object {
  "output": "another complicated task output for input bar",
}
`;

exports[`The complicatedLegacyModule doComplicatedTask() function works as expected 1`] = `
Object {
  "output": "complicated task output for input foo",
}
`;
